tests/test_database_types.py

```diff
@@ -178,60 +242,135 @@ def expand_params(testcase_func, param_num, param):
     )
 
 
-def _insert_to_table(conn, table, values):
-    insertion_query = f"INSERT INTO {table} (id, col) VALUES "
-    for j, sample in values:
-        insertion_query += f"({j}, '{sample}'),"
-
-    conn.query(insertion_query[0:-1], None)
-    if not isinstance(conn, db.BigQuery):
-        conn.query("COMMIT", None)
-
-
 class TestDiffCrossDatabaseTables(unittest.TestCase):
+    # https://docs.python.org/2/library/unittest.html#unittest.TestCase.maxDiff
+    # For showing assertEqual differences under a certain length.
+    maxDiff = 10000
+
     @parameterized.expand(type_pairs, name_func=expand_params)
     def test_types(self, source_db, target_db, source_type, target_type, type_category):
-        start = time.time()
-
         self.src_conn = src_conn = CONNS[source_db]
         self.dst_conn = dst_conn = CONNS[target_db]
 
         self.connections = [self.src_conn, self.dst_conn]
         sample_values = TYPE_SAMPLES[type_category]
 
-        src_table_path = src_conn.parse_table_name("src")
-        dst_table_path = dst_conn.parse_table_name("dst")
+        # We need to include the database name because of e.g. Presto which has
+        # Postgres as a backing catalog and shouldn't re-use those.
+        src_table_name = f"src_{source_db.__name__.lower()}_{parameterized.to_safe_name(source_type)}_{N_SAMPLES}"
+        src_table_path = src_conn.parse_table_name(src_table_name)
         src_table = src_conn.quote(".".join(src_table_path))
+
+        dst_table_name = f"dst_{target_db.__name__.lower()}_{parameterized.to_safe_name(target_type)}_{src_table_name}"
+        # Since we insert `src` to `dst`, the `dst` can be different for the
+        # same type. 😭
+        dst_table_path = dst_conn.parse_table_name(dst_table_name)
         dst_table = dst_conn.quote(".".join(dst_table_path))
 
+        # If you want clean house from e.g. changing fakers.
         src_conn.query(f"DROP TABLE IF EXISTS {src_table}", None)
```

Comment:
Oracle doesn't support "DROP TABLE IF EXISTS"

Reply:
I'm actually going to remove/comment out these lines. The whole idea is to keep these tables around so we can re-run benchmarks without building up millions of rows.